frontend/interfaces/platform.ts

```diff
@@ -73,5 +73,7 @@ export const HOST_LINUX_PLATFORMS = [
  * the possible Linux-like platform values.
  */
 export const isLinuxLike = (platform: string) => {
-  return HOST_LINUX_PLATFORMS.includes(platform as any);
+  return HOST_LINUX_PLATFORMS.includes(
+    platform as typeof HOST_LINUX_PLATFORMS[number]
```

Comment:
would it make sense to remove the above `as const`, allowing `HOST_LINUX_PLATORMS` to be `string[]` so we can leave `platform` typed as `string` as well?

Reply:
I get the casting in the typeguarding case but it seems unnecessary here.

Reply:
@jacobshandling, my thinking is that we definitely want `HOST_LINUX_PLATFORMS` to be `as const` so that TypeScript alerts us if any code tries to mutate that object. The type assertion makes the `includes` syntax a bit more verbose here, but I think it is the best and safest way to do it. In the event the caller provides a `platform` that is not actually of the asserted type, it certainly won't exist in the array and we can be sure that `includes` will return `false` just as we expect.